de.zray.Facetracker.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app-id": "de.zray.Facetracker",
```

Comment:
there is no indication that `zray.de` is associated to the upstream developers. There is only a parking page.

Reply:
Oh my bad, are dashes allowed as appids? The actual URL is https://z-ray.de

Reply:
See the rules https://docs.flathub.org/docs/for-app-authors/requirements/#application-id

Reply:
Missed that, many thanks will fix.